File: 832.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        for i in range(len(image)):
            for j in range(len(image[i])//2):
                temp=image[i][j]
                image[i][j]=image[i][len(image[i])-1-j]
                image[i][len(image[i])-1-j]=temp
        
        for i in range(len(image)):
            for j in range(len(image[i])):
                if image[i][j]:
                    image[i][j]=0
                else:
                    image[i][j]=1
        return image                    

File: 832.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        for(int i=0;i<image.size();i++)
        {
            for(int j=0;j<image[i].size()/2;j++)
            {
                int temp;
                temp=image[i][j];
                image[i][j]=image[i][image[i].size()-1-j];
                image[i][image[i].size()-1-j]=temp;
            }
        }
        
        for(int i=0;i<image.size();i++)
        {
            for(int j=0;j<image[i].size();j++)
            {
                if (image[i][j])
                {
                    image[i][j]=0;
                }
                else
                {
                    image[i][j]=1;
                }
            }
        }
        return image;
    }
};
